<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>[639] Logout error</title>
    <link>https://forum.moparisthebest.com/t/639-logout-error/448344</link>
    <description>When someone on my server logs out, it lags the ENTIRE server for about 5-10 seconds and I get this error in my run.bat
[code]at java.lang.Thread.run(Unknown Source)
com.mysql.jdbc.CommunicationsException: Communications link failure due to under
lying exception:

** BEGIN NESTED EXCEPTION **

java.net.ConnectException
MESSAGE: Connection refused: connect

STACKTRACE:

java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
        at java.net.PlainSocketImpl.connect(Unknown Source)
        at java.net.SocksSocketImpl.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja
va:173)
        at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:267)
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)
        at org.dementhium.mysql.Highscores.query(Highscores.java:45)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)


** END NESTED EXCEPTION **



Last packet sent to the server was 0 ms ago.
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)
        at org.dementhium.mysql.Highscores.query(Highscores.java:45)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
        at org.dementhium.mysql.Highscores.query(Highscores.java:40)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
        at org.dementhium.mysql.Highscores.destroyConnection(Highscores.java:53)

        at org.dementhium.mysql.Highscores.query(Highscores.java:44)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
com.mysql.jdbc.CommunicationsException: Communications link failure due to under
lying exception:

** BEGIN NESTED EXCEPTION **

java.net.ConnectException
MESSAGE: Connection refused: connect

STACKTRACE:

java.net.ConnectException: Connection refused: connect
        at java.net.DualStackPlainSocketImpl.connect0(Native Method)
        at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
        at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
        at java.net.PlainSocketImpl.connect(Unknown Source)
        at java.net.SocksSocketImpl.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.connect(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at java.net.Socket.&lt;init&gt;(Unknown Source)
        at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja
va:173)
        at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:267)
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)
        at org.dementhium.mysql.Highscores.query(Highscores.java:45)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)


** END NESTED EXCEPTION **



Last packet sent to the server was 0 ms ago.
        at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)
        at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
        at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java
:266)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)
        at org.dementhium.mysql.Highscores.query(Highscores.java:45)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
        at org.dementhium.mysql.Highscores.query(Highscores.java:40)
        at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2013)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
        at org.dementhium.mysql.Highscores.destroyConnection(Highscores.java:53)

        at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
ctionButtonHandler.java:2014)
        at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac
tionButtonHandler.java:51)
        at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)
        at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu
mHandler.java:28)
        at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2
3)
        at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas
kExecutor.java:13)
        at org.dementhium.model.World.run(World.java:177)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)

        at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
access$301(Unknown Source)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)[/code]
Any idea what&#39;s wrong?</description>
    
    <lastBuildDate>Thu, 05 Jul 2012 17:03:53 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/639-logout-error/448344.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>[639] Logout error</title>
        <dc:creator><![CDATA[@99rangek0 99rangek0]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/99rangek0">@99rangek0</a> wrote:</p>
          <blockquote>
              <p>[quote=“AtomicInt_, post:3, topic:448344”]When the user logs out you’re creating a SQL connection to a data base that is not there.</p>
<p>Probably server high scores or something.[/quote]</p>
<aside class="quote" data-post="2" data-topic="448344">
<div class="title">
<div class="quote-controls"></div>
<img alt width="20" height="20" src="https://forum.moparisthebest.com/letter_avatar/dannyissopro/40/5_e05bb34c421432ee4d40de30c10af3e5.png" class="avatar"> DannyIsSoPro:</div>
<blockquote>
<p>From the looks of it, it looks like the server is failing to connect to the MYSQL database. Nothing to worry about, just remove the MYSQL.</p>
</blockquote>
</aside>
<p>Hey, thanks both of you guys it worked :D.<br>
Hadn’t even considered the SQL I tend to overthink things <img src="https://forum.moparisthebest.com/images/emoji/twitter/stuck_out_tongue.png?v=5" title=":stuck_out_tongue:" class="emoji" alt=":stuck_out_tongue:"> haha<br>
well anyways, thanks!</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/639-logout-error/448344/4">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/639-logout-error/448344/4</link>
        <pubDate>Thu, 05 Jul 2012 17:03:53 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-448344-4</guid>
        <source url="https://forum.moparisthebest.com/t/639-logout-error/448344.rss">[639] Logout error</source>
      </item>
      <item>
        <title>[639] Logout error</title>
        <dc:creator><![CDATA[@ExtremeX-Scape Ryley]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/extremex-scape">@ExtremeX-Scape</a> wrote:</p>
          <blockquote>
              <p>When the user logs out you’re creating a SQL connection to a data base that is not there.</p>
<p>Probably server high scores or something.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/639-logout-error/448344/3">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/639-logout-error/448344/3</link>
        <pubDate>Thu, 05 Jul 2012 06:08:51 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-448344-3</guid>
        <source url="https://forum.moparisthebest.com/t/639-logout-error/448344.rss">[639] Logout error</source>
      </item>
      <item>
        <title>[639] Logout error</title>
        <dc:creator><![CDATA[@DannyIsSoPro DannyIsSoPro]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/dannyissopro">@DannyIsSoPro</a> wrote:</p>
          <blockquote>
              <p>From the looks of it, it looks like the server is failing to connect to the MYSQL database. Nothing to worry about, just remove the MYSQL.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/639-logout-error/448344/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/639-logout-error/448344/2</link>
        <pubDate>Thu, 05 Jul 2012 04:58:37 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-448344-2</guid>
        <source url="https://forum.moparisthebest.com/t/639-logout-error/448344.rss">[639] Logout error</source>
      </item>
      <item>
        <title>[639] Logout error</title>
        <dc:creator><![CDATA[@99rangek0 99rangek0]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/99rangek0">@99rangek0</a> wrote:</p>
          <blockquote>
              <p>When someone on my server logs out, it lags the ENTIRE server for about 5-10 seconds and I get this error in my run.bat</p>
<p>[code]at java.lang.Thread.run(Unknown Source)<br>
com.mysql.jdbc.CommunicationsException: Communications link failure due to under<br>
lying exception:</p>
<p>** BEGIN NESTED EXCEPTION **</p>
<p>java.net.ConnectException<br>
MESSAGE: Connection refused: connect</p>
<p>STACKTRACE:</p>
<p>java.net.ConnectException: Connection refused: connect<br>
at java.net.DualStackPlainSocketImpl.connect0(Native Method)<br>
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)<br>
at java.net.PlainSocketImpl.connect(Unknown Source)<br>
at java.net.SocksSocketImpl.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja<br>
va:173)<br>
at com.mysql.jdbc.MysqlIO.(MysqlIO.java:267)<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:45)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)</p>
<p>** END NESTED EXCEPTION **</p>
<p>Last packet sent to the server was 0 ms ago.<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:45)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)<br>
java.lang.NullPointerException<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:40)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:67)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)<br>
java.lang.NullPointerException<br>
at org.dementhium.mysql.Highscores.destroyConnection(Highscores.java:53)</p>
<pre><code>    at org.dementhium.mysql.Highscores.query(Highscores.java:44)
    at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)
    at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
</code></pre>
<p>ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)<br>
com.mysql.jdbc.CommunicationsException: Communications link failure due to under<br>
lying exception:</p>
<p>** BEGIN NESTED EXCEPTION **</p>
<p>java.net.ConnectException<br>
MESSAGE: Connection refused: connect</p>
<p>STACKTRACE:</p>
<p>java.net.ConnectException: Connection refused: connect<br>
at java.net.DualStackPlainSocketImpl.connect0(Native Method)<br>
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)<br>
at java.net.PlainSocketImpl.connect(Unknown Source)<br>
at java.net.SocksSocketImpl.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.ja<br>
va:173)<br>
at com.mysql.jdbc.MysqlIO.(MysqlIO.java:267)<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:45)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)</p>
<p>** END NESTED EXCEPTION **</p>
<p>Last packet sent to the server was 0 ms ago.<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java<br>
:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at org.dementhium.mysql.Highscores.createConnection(Highscores.java:28)<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:45)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)<br>
java.lang.NullPointerException<br>
at org.dementhium.mysql.Highscores.query(Highscores.java:40)<br>
at org.dementhium.mysql.Highscores.saveHighScore(Highscores.java:68)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A<br>
ctionButtonHandler.java:2013)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)<br>
java.lang.NullPointerException<br>
at org.dementhium.mysql.Highscores.destroyConnection(Highscores.java:53)</p>
<pre><code>    at org.dementhium.net.packethandlers.ActionButtonHandler.handleButtons(A
</code></pre>
<p>ctionButtonHandler.java:2014)<br>
at org.dementhium.net.packethandlers.ActionButtonHandler.handlePacket(Ac<br>
tionButtonHandler.java:51)<br>
at org.dementhium.net.PacketManager.handlePacket(PacketManager.java:54)<br>
at org.dementhium.net.handler.DementhiumHandler.processPackets(Dementhiu<br>
mHandler.java:28)<br>
at org.dementhium.task.impl.PlayerTickTask.execute(PlayerTickTask.java:2<br>
3)<br>
at org.dementhium.task.SequentialTaskExecutor.performTasks(SequentialTas<br>
kExecutor.java:13)<br>
at org.dementhium.model.World.run(World.java:177)<br>
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)<br>
at java.util.concurrent.FutureTask$Sync.innerRunAndReset(Unknown Source)</p>
<pre><code>    at java.util.concurrent.FutureTask.runAndReset(Unknown Source)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.
</code></pre>
<p>access$301(Unknown Source)<br>
at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.<br>
run(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)<br>
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)<br>
at java.lang.Thread.run(Unknown Source)[/code]<br>
Any idea what’s wrong?</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/639-logout-error/448344/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/639-logout-error/448344/1</link>
        <pubDate>Thu, 05 Jul 2012 04:29:57 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-448344-1</guid>
        <source url="https://forum.moparisthebest.com/t/639-logout-error/448344.rss">[639] Logout error</source>
      </item>
  </channel>
</rss>
